Neo4j.Driver/Neo4j.Driver/Internal/Connector/TcpSocketClient.cs

```diff
@@ -34,7 +34,8 @@ internal class TcpSocketClient : ITcpSocketClient
         public TcpSocketClient(EncryptionManager encryptionManager, bool keepAlive, ILogger logger = null)
         {
             _encryptionManager = encryptionManager;
-            _client = new TcpClient();
+            _client = new TcpClient(AddressFamily.InterNetworkV6);
```

Comment:
Shall considering to add a configuration option to choose to whether turn on ipv6 or not.